### ISSUE TITLE
supporting code samples for multipart file upload

### ISSUE THREAD

**Issue by cwcowell** [fern-api/fern#5406]
### CLI Version (from `fern.config.json`)

0.46.1

### Minimal API Specification

```yaml
openapi: 3.0.0
info:
  title: Demo API
  version: "0.1"
paths:
  /v2/conversations:
    post:
      operationId: createConversation
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                files:
                  type: array
                  description: Files with local file paths
                  items:
                    type: string
                    format: binary
      responses:
        '200':
          description: OK
      x-fern-examples:
        - code-samples:
            - language: curl
              code: curl https://yahoo.com/
            - language: python
              code: print("hello”)
```

### Actual UI Behavior

There’s no dropdown to pick the language for the sample code, and the sample code provided in `openapi.yaml` is ignored. Fern-generated code is shown instead.

<img width="1119" alt="image" src="https://github.com/user-attachments/assets/2af8a876-7ae0-4233-ab64-ac01f3be88da" />

### Expected UI Behavior

There should be a dropdown to pick the language for the sample code, and the sample code provided in `openapi.yaml` should be displayed.

### Steps to Reproduce

Start up a docs site with this `openapi.yaml` and visit the API reference page for `Conversations`/`post`.

### Environment Details

Chrome 131, Safari 18.1.1, macOS 15.1

**Comment by cwcowell** [comment]
This is a **high-priority** issue for us, since we rely on hand-written, heavily commented SDK sample code in the API reference pages to serve as the main reference material for our SDK.

**Comment by dsinghvi** [comment]
@cwcowell this should be fixed on the latest CLI version!

**Comment by cwcowell** [comment]
Yep, confirmed this works for us now. Thanks for getting that fixed.
